gatsby-site/src/components/About.js

```diff
@@ -43,8 +27,7 @@ const Paragraph = styled.p`
  */
 const About = ({ hubName, content }) => {
   return (
-    <Section>
-      <Heading>About Sunrise {hubName}</Heading>
+    <Section id="about" title={`About Sunrise ${hubName}`}>
```

Comment:
Just want to flag that there might be hubs whose names don't start with `Sunrise ` -- the `name` attribute in the JSON should contain the right info, though!

Reply:
Will follow up on this in another PR